src/domains/citizen-auth-app/07_function_fast_login.tf

```diff
@@ -133,6 +134,26 @@ module "function_fast_login" {
     "blobs_retention_days"       = 0,
   }
 
+  storage_account_info = {
+    account_kind                      = "StorageV2"
+    account_tier                      = "Standard"
+    account_replication_type          = "ZRS"
+    access_tier                       = "Hot"
+    advanced_threat_protection_enable = true
+    use_legacy_defender_version       = true
+    public_network_access_enabled     = true
```

Comment:
is this right? (publicily accessible storage?)

Reply:
Actually, it can be removed because it's hardcoded to true in the module itself:

```hcl
module "storage_account" {
  source = "github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v7.76.0"

  name                          = coalesce(var.storage_account_name, format("%sst", replace(var.name, "-", "")))
  account_kind                  = var.storage_account_info.account_kind
  account_tier                  = var.storage_account_info.account_tier
  account_replication_type      = var.storage_account_info.account_replication_type
  access_tier                   = var.storage_account_info.account_kind != "Storage" ? var.storage_account_info.access_tier : null
  resource_group_name           = var.resource_group_name
  location                      = var.location
  advanced_threat_protection    = var.storage_account_info.advanced_threat_protection_enable
  use_legacy_defender_version   = var.storage_account_info.use_legacy_defender_version
  public_network_access_enabled = true

  tags = var.tags
}
```